DSCResources/MSFT_xDnsServerForwarder/MSFT_xDnsServerForwarder.schema.mof

```diff
@@ -0,0 +1,7 @@
+[ClassVersion("1.0.0"), FriendlyName("xDnsServerForwarder")]
+class MSFT_xDnsServerForwarder :  OMI_BaseResource
+{
+    [Key] String IPAddress;
+    [Write] Boolean RemoveAll;
```

Comment:
@TravisEz13 It should be RemoveAll. If this property is $true, it ignores the IPAddress property because it will just remove all forwarders currently set on the DNS server if any exist.

From my experience, DSC isn't flexible enough to allow parameter sets for Key properties so that only one or the other is allowed. See the updated README.md for examples of each scenario.